jahia-cloud-env-update/update-to-2.1.0/update.yml

```
@@ -27,3 +27,10 @@ actions:
         systemctl daemon-reload
         exit $?
       user: root
+
+  healthcheckOnHaproxy:
+    - cmd[bl]: |-
+        cp /etc/haproxy/haproxy.cfg.d/00-global.cfg /root/00-global.cfg.bak
+        sed -ie '/\!tools #HTTP_AUTH_BASIC/a acl healthcheck path /healthcheck\n    http-request allow if healthcheck' /etc/haproxy/haproxy.cfg.d/00-global.cfg
```

Update scripts are attended to be apply version by version so using line number should'nt be a problem but ok for that. But at least having line return and right indentation like in the [related PR](https://github.com/Jahia/paas_jelastic_dx_universal/pull/117/files) will be cool ^_^
```suggestion
        sed -ie '/\!tools #HTTP_AUTH_BASIC/a\\n    acl healthcheck path /healthcheck\n    http-request allow if healthcheck' /etc/haproxy/haproxy.cfg.d/00-global.cfg
```

---

I agree, I tried but i couldn't make it work... removing space between a and \\n seemed wrong to me I didn't tried it, thanks.
Fixed